tools/build_config/clone_herbert_branch.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
unlike the powershell script, this one will leave the current directory as `[...]/Herbert`

Reply:
is there any reason we don't have an "invoke-in-dir" equivalent function for all the bash scripting?